rancher/rancher-ui/infrastructure/hosts/custom/index.md

```diff
@@ -40,9 +40,9 @@ http://<rancher-server-ip>:8080/v1/projects/1a5/scripts/<registrationToken>
 
 ### Security Groups/Firewalls 
 
-For any hosts that are added, ensure that any security groups or firewalls allow traffic. If these are not enabled, the Rancher functionality will be limited.
+For any hosts that are added, ensure that any security groups or firewalls allow traffic between the Rancher server and the Rancher agents, otherwise Rancher functionality will be limited.
 
-* From and to all other hosts on UDP ports `500` and `4500` (for IPsec networking)
+* From Rancher server to UDP ports `500` and `4500` (for IPsec networking) on Rancher agent hosts
 
```

Comment:
Yes, that is what your PR says, and that's why it's wrong.  If you have 4 machines A-D running 

A: rancher/server
B: rancher/agent
C: rancher/agent
D rancher/agent

You need 
- 500 and 4500/udp in both directions between all combinations B, C, and D.  
- 8080/tcp from B, C, and D to A

A is what people the call the "Rancher server" and does NOT make or receive any IPSec connections.